### DISCUSSION TITLE
Stopping codespace

### DISCUSSION THREAD

**Discussion by JKinsler** [earthlab-education/Earth-Analytics-2023-01-Intro#242] [! GitHub help] [answered]
Hi, I was completing the 'Temperature over Time' assignment when my Codespace suddenly stopped. When I try to re-enter the codespace I get the same error message 'Stopping Codespace'. Any suggestions?

**Comment by nquarder** [comment]
Hi @JKinsler can we try to take a look at this during discussion today? I don't know that I've encountered this before but hopefully we can get it figured out w/o too much hassle. -NQ

**Comment by edchanonline** [comment, marked as answer]
I ran into that briefly, and I went to https://github.com/codespaces/ to start the codespace again. I was afraid I would have lost all my work since I didn't make any commits but it seemed the constainer stopped and preserved the state so hopefully that's your case.

**Reply by eculler** [reply]
Marking this as the answer for now - do this work for anyone else having this problem? I've not run into it myself either.

**Comment by misterskye** [comment]
I kind of had a similar scenario last night, but different of course...  My Codespace, "Super Robot" stopped connecting when I try to bring it up, and so I created a new one and started somewhat fresh (with some angst).
